docs/data-export.md

```diff
@@ -1,18 +1,35 @@
 # Data Export
 
-### How to Export Data from Scarf
+## Introduction
 
-To export data out of Scarf, go to the main dashboard and click "Export packages data".
+Scarf provides a robust platform for tracking package downloads and pixel views. The ability to export this data is crucial for analytics, reporting, and integrating with other tools. This guide aims to provide a clear and concise explanation of how to export data from Scarf, what data is exported, and how to make use of any available integrations.
+
+## Prerequisites
+
+Exporting data from Scarf will only work if you are on a [Scarf Premium Tier plan](https://about.scarf.sh/#pricing-section).
```

Comment:
Should link to https://about.scarf.sh/#pricing now, and should maybe say "Scarf paid-tier plan"? Since both Basic and Premium allow you to export

Reply:
Updated